### PR TITLE
docs: Add Code of Conduct

### LINKED ISSUE #454: [docs] Add Code of Conduct
There is an in progress TDR to adopt the[ Contributor Covenant](https://www.contributor-covenant.org/version/2/0/code_of_conduct/) for all Truss projects. We should do this anyway.  

As stated in the TDR "By adopting a code of conduct, we signal the intention for our open source community to align with Truss’ goal of being welcoming, diverse, inclusive, and equitable."

### PR DESCRIPTION
**Pull request by haworku**
# Summary

## Related Issues or PRs
closes #454 

[This PR may have to hold until email alias is created and/or TDR is approved]

### CODE REVIEW

**Review by suzubara — APPROVED** [review]
🎉

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official e-mail address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at
+[INSERT CONTACT METHOD].
```
Comment (brainsik) [inline CODE_OF_CONDUCT.md]: This needs to get updated with a contact method. As part of the [TDR-0073](https://docs.google.com/document/d/1YYTM92O0xRgiv_FwToeqnlqvhWqmLJoTi35NuXVaAj8/edit) approval I'll be setting up an email alias to receive these. However, if you have a contact address for the React USWDS project, feel free to use that.

**Review by brainsik — DISMISSED** [review]

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (brainsik) [reply CODE_OF_CONDUCT.md]: Thanks for tagging me, @haworku!

Comment (haworku) [reply CODE_OF_CONDUCT.md]: @brainsik yep I had already filled in oss-coc@truss.works in 
6aa4485.   Can you ping in the oss slack channel when that is set up? We don't have an email address for the project so gonna let this PR sit open until that alias is created.

Comment (brainsik) [reply CODE_OF_CONDUCT.md]: Sorry I missed that. Will do!

**Comment by haworku** [comment]
Confirmed that oss-coc@truss.works is up

**Comment by brainsik** [comment]
> Confirmed that [oss-coc@truss.works](mailto:oss-coc@truss.works) is up

The email address is setup, but we agreed to create docs about how to do the enforcement before publishing. That work is being tracked as [InfraSec #174849673](https://www.pivotaltracker.com/story/show/174849673).

**Pull request merged**